src/quickpicks/common.ts

```diff
@@ -52,11 +52,11 @@ export async function inputBox (options: InputBoxOptions) {
 	return input;
 }
 
-export async function quickPick (items: any[], options?: QuickPickOptions) {
-	if (items.length === 1) {
+export async function quickPick (items: any[], quickPickOptions?: QuickPickOptions, options?: any) {
+	if (items.length === 1 && !options.forceShow) {
```

Comment:
If `options` is optional we need a check for existence first before accessing the prop.

It may be better to make `options` have a default of `{ forceShow: true }` similar to in `runCommand` in `terminal.ts`